appcontainer/AppStateModel.ts

```diff
@@ -94,9 +94,13 @@ export class AppStateModel extends HoistModel {
                     data: {
                         appVersion: XH.appVersion,
                         appBuild: XH.appBuild,
+                        clientId: XH.clientId,
+                        sessionId: XH.sessionId,
                         locationHref: window.location.href,
```

Comment:
Minor thing, but looks like `locationHref` might be legacy, from before we capture this by default on the top-level track entry (outside of data)